integration_tests/features/support/steps.js

```diff
@@ -220,3 +221,45 @@ Then(/(.*) should have (\d+) peers/, async function (nodeName, peerCount){
     // we add a non existing node when the node starts before adding any actual peers. So the count should always be 1 higher
     expect(peers.length).to.equal(peerCount+1)
 })
+
+When(/I send (.*) tari from (.*) to (.*),(.*) at fee (.*)/, async function (tariAmount,source,dest,dest2,fee) {
+ let wallet = this.getWallet(source);
+ let client = wallet.getClient();
+ var destWallet = this.getWallet(dest);
+ var destWallet2 = this.getWallet(dest2);
+ console.log("Pubkey:", destWallet.getPubKey());
+ console.log("Pubkey:", destWallet2.getPubKey());
+ let output = await client.transfer({"recipients": [{"address": destWallet.getPubKey(),
+                                                     "amount": tariAmount,
+                                                      "fee_per_gram": fee,
+                                                      "message": "msg"
+                                                     },
+                                                     {
+                                                      "address": destWallet2.getPubKey(),
+                                                      "amount": tariAmount,
+                                                      "fee_per_gram": fee,
+                                                      "message": "msg"}]
+                                                    });
+  console.log("output", output);
+  result = output;
+});
+
+When(/I wait (.*) seconds/, async  function (int) {
```

Comment:
This test will always succeed in the amount of time given, so you can add a very large timeout to enable to user to put any value in
```suggestion
When(/I wait (.*) seconds/, {timeout: 600*1000}, async  function (int) {
```